src/proto/chk/chk.proto

```diff
@@ -193,7 +193,11 @@ message CheckReport {
         uint64 seq = 1; // DAOS Check event sequence, unique for the instance.
         CheckInconsistClass class = 2; // Inconsistency class
         CheckInconsistAction action = 3; // The action taken to repair the inconsistency
-        bool repaired = 4; // Repaired or not, meaningless if the action is CIA_INTERACT.
+        // Repair result: zero is for repaired successfully.
+        //		  negative value if failed to repair.
+        //		  positive value is for CIA_IGNORE or dryrun mode.
+        // It is meaningless if the action is CIA_INTERACT.
+        int32 result = 4;
```

Comment:
How is this different from field #3, `action`? Seems like there is some redundancy here. If not, then it seems we should set the field type to an enum to constrain the set of possible result values, no?

Reply:
"CheckInconsistAction action = 3" means how do we (try to) handle the inconsistency, for example, remove the dangling reference, or re-reference the orphan shard. The "int32 result = 4;" means the action result (succeed, or fail for some reason).

Reply:
OK, so is it a boolean value? If so, perhaps it should be changed to `bool succeeded = 4;`? If not, and there is a closed set of possible values, then it should probably be an enum, right? If it is an open-ended set of possible values, how do we know what should be displayed to the user?